Travis-CI: fix macOS builds

tschoonj opened the pull request.

tschoonj committed:
Travis-CI: fix macOS builds

tschoonj commented:
Travis-CI werkt nu, AppVeyor is next. Zal dat doen in deze pull request dus openhouden aub.

tschoonj committed:
Add AppVeyor support

tschoonj commented:
@PieterTack Kan je eens checken of ik effectief ben toegevoegd als collaborator in de settings? Thanks!

PieterTack commented:
https://github.com/PieterTack/polycap/invitations
Volgens de site staat mijn request nog als active. Mss werkt bovenstaande link?

---
X-Ray Micro-Spectroscopy and Imaging Group
Department of Chemistry
Ghent University
Krijgslaan 281 S12
B-9000 Ghent
Belgium
Phone: +32 (0)9 264 4723
Fax: +32 (0)9 264 49 60

________________________________________
Van: Tom Schoonjans [notifications@github.com]
Verzonden: vrijdag 22 december 2017 10:43
Aan: PieterTack/polycap
CC: Pieter Tack; Mention
Onderwerp: Re: [PieterTack/polycap] Travis-CI: fix macOS builds (#3)

@PieterTack<https://github.com/pietertack> Kan je eens checken of ik effectief ben toegevoegd als collaborator in de settings? Thanks!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/PieterTack/polycap/pull/3#issuecomment-353561597>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AhIfPD9zwBwzNcTw9fWDTNFk6C1Ks9Q-ks5tC3nRgaJpZM4RKwMj>.

tschoonj commented:
Sorry, ik had de email niet gezien!

tschoonj committed:
Remove random.dat dependency

Do not depend on a seed that would be fetched/stored in a file.
Instead use /dev/urandom on Linux/macOS and rand_s on Windows to obtain
cryptographically secure, high quality seeds.

tschoonj commented:
Hey Pieter,

Ik heb er nog een commit bijgedaan die het gebruik van `random.dat` overbodig maakt. Als je het goed vindt, mag je de pull request mergen.
